rust/headless-client/src/imp_linux.rs

```diff
@@ -185,9 +185,9 @@ pub(crate) fn run_ipc_service(cli: Cli) -> Result<()> {
 }
 
 pub fn firezone_group() -> Result<nix::unistd::Group> {
-    let group = nix::unistd::Group::from_name("firezone")
+    let group = nix::unistd::Group::from_name(FIREZONE_GROUP)
         .context("can't get group by name")?
-        .context("firezone group must exist on the system")?;
+        .context("`{FIREZONE_GROUP}` group must exist on the system")?;
```

Comment:
@ReactorScram This string interpolation does not work, you need `format!` for that :)